src/Markup/Avalonia.Markup/Data/MultiBinding.cs

```diff
@@ -116,7 +116,13 @@ private object ConvertValue(IList<object> values, Type targetType, IMultiValueCo
             }
 
             var culture = CultureInfo.CurrentCulture;
-            var converted = converter.Convert(values, targetType, ConverterParameter, culture);
+            object converted;
+            if (converter != null)
+            {
+                converted = converter.Convert(values, targetType, ConverterParameter, culture);
+            }
+            else
+                converted = values.ToArray();
```

Comment:
Is there a reason why you are copying this list?

Reply:
I don't want to allow values.add to be used for consistency reasons